tcp.go

```diff
@@ -11,19 +12,34 @@ import (
 	"github.com/prometheus/log"
 )
 
+func dialTCP(target string, ssl bool, timeout time.Duration) (net.Conn, error) {
+	dialer := &net.Dialer{Timeout: timeout}
+	if !ssl {
+		return dialer.Dial("tcp", target)
+	}
+	config := &tls.Config{InsecureSkipVerify: true}
```

Comment:
This needs to be secure by default.

Similar to #23, if we're going to be offering non-default SSL options they should be common across prometheus/blackbox_exporter/alertmanger.